### ISSUE TITLE
Search keyword or text in Chrome or default Browser

### ISSUE THREAD

**Issue by lRadha** [sumatrapdfreader/sumatrapdf#1856]
Please add a context-menu feature to search text on the browser.

**Comment by GitHubRulesOK** [comment]
There is a PR pending for such feature, and it is possible to "brew your own" by scripting several different ways, e.g. using cmd+ or vbs or ahk or any 3rdparty selection monitor.
Once selections are exposed they can be manipulated much more than just "search", since they can then be used to "look-up", translate, speak out loud or any other action.

In this screenshot I selected the word "commandline" and pressed Ctrl+4 (could be any other key I chose) and a simple AutoHotKey script then invokes default browser (internet explorer non-private mode with duckduckgo)
However I have 20 other ways I can call desired browsers (see inset) which is just the major browsers (there are many many others) and each may have different syntax, hence there is a question as to what each user needs as their "default" browser is it inprivate or incognito mode, is it for search / translate, speaking out loud or simply viewing search result in another application.
![image](https://user-images.githubusercontent.com/12754913/104173466-b72dde80-53fd-11eb-8ce0-0b6629d952b5.png)
 here is a more complex one that reads out loud /  searches
![image](https://user-images.githubusercontent.com/12754913/104180816-aaad8400-5405-11eb-8d32-d4e57d958a93.png)

For a simpler vbs method that should work on most versions of windows see https://forum.sumatrapdfreader.org/t/advanced-tips-pass-selection-to-another-app-e-g-web-search/2700
